index.ts

```diff
@@ -0,0 +1,33 @@
+import express, { Express } from 'express';
+import mongoose from 'mongoose';
+import multer from 'multer';
+
+import { checkRole } from './src/admin/admin.controller';
+import fileRouter from './src/file/file.route';
+import {
+  dbName,
+  port,
+  uriMongoDb,
+  multerFieldName,
+  multerDestinationFolder,
+} from './common/constants';
+
+const app: Express = express();
+
+app.use(express.urlencoded({ extended: false }));
+app.use(express.json());
+
+app.use('/*/:uuid', checkRole);
```

Comment:
Please, move checkRole function to the middleware for the fileRouter rotes. Or use this function as a global moddleware.